sample-root-scripts/read_LIGen_OD_output.C

```diff
@@ -0,0 +1,335 @@
+// C++ Includes
+#include <iostream>
+//#include <algorithm>
+//#include <iomanip>
+#include <cmath>
+
+// ROOT Includes
+#include "TH1F.h"
+#include "TH2F.h"
+#include "TFile.h"
+//#include "TApplication.h"
+
+
+// WCSim Includes
+#include "WCSimRootEvent.hh"
+#include "WCSimRootGeom.hh"
+
+// Defines
+#define PI 3.141592654
+#define topPMT 19
+/*
+*How to run:
+*
+* enter in the terminal $WCSIMDIR/rootwc/rootwc -l 'read_OD.C("WCSim.root","outputFile.root",false)' to run the code
```

Comment:
Please update the how to run with the correct filename

Also it would be nice to have a quick "this is what this macro does" comment